dipy/reconst/sfm.py

```diff
@@ -147,20 +286,27 @@ def __init__(self, gtab, sphere=None, response=[0.0015, 0.0005, 0.0005],
             The eigenvalues of a canonical tensor to be used as the response
             function of single-fascicle signals.
             Default:[0.0015, 0.0005, 0.0005]
-
-        solver : string or SKLearnLinearSolver object, optional
+        solver : string, dipy.core.optimize.SKLearnLinearSolver object, or
+            sklearn.linear_model.base.LinearModel object, optional.
```

Comment:
This works.

On Thu, Mar 12, 2015 at 7:44 PM, Ariel Rokem <arokem@gmail.com> wrote:

> Good catch.
>
> On Thu, Mar 12, 2015 at 7:27 PM, Stefan van der Walt <
> notifications@github.com> wrote:
>
>> In dipy/reconst/sfm.py
>> <https://github.com/nipy/dipy/pull/559#discussion_r26361310>:
>>
>> > @@ -147,20 +286,27 @@ def __init__(self, gtab, sphere=None, response=[0.0015, 0.0005, 0.0005],
>> >              The eigenvalues of a canonical tensor to be used as the response
>> >              function of single-fascicle signals.
>> >              Default:[0.0015, 0.0005, 0.0005]
>> > -
>> > -        solver : string or SKLearnLinearSolver object, optional
>> > +        solver : string, dipy.core.optimize.SKLearnLinearSolver object, or
>> > +            sklearn.linear_model.base.LinearModel object, optional.
>>
>> Does this render correctly?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/nipy/dipy/pull/559/files#r26361310>.
>>
>
>